multisig-factory/src/lib.rs

```diff
@@ -0,0 +1,50 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::json_types::Base58PublicKey;
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::serde_json::json;
+use near_sdk::{env, near_bindgen, AccountId, Promise};
+
+#[global_allocator]
+static ALLOC: near_sdk::wee_alloc::WeeAlloc<'_> = near_sdk::wee_alloc::WeeAlloc::INIT;
+
+const CODE: &[u8] = include_bytes!("../../multisig2/res/multisig.wasm");
```

Comment:
```suggestion
const CODE: &[u8] = include_bytes!("../../multisig2/res/multisig2.wasm");
```
See:
https://github.com/near/core-contracts/pull/125/files#diff-563ebbdd4d50f30899977aebf36f53bcda8646c5bb5f5c6804c615b73a634f53R5
and manifest:
https://github.com/near/core-contracts/pull/125/files#diff-d40e917a59bc0f7f77b1e0e76ad1de4f7052fc19428233acdb578caedd69296cR2